diff --git a/build/SymbolView.js b/build/SymbolView.js
index 6b352b5bef63ede4877eee3a8ed2cae359c5ebdf..201dfa41474911a49543d4a38baf99ddc6239b69 100644
--- a/build/SymbolView.js
+++ b/build/SymbolView.js
@@ -32,7 +32,14 @@ export function SymbolView(props) {
     if (!loaded) {
         return _jsx(View, { style: style });
     }
-    return (_jsx(View, { style: style, children: _jsx(Text, { style: {
+    return (_jsx(View, { style: style, children: _jsx(Text, { allowFontScaling: Platform.OS === 'android' ? false : undefined, style: {
+                ...(Platform.OS === 'android' && {
+                    width: '100%',
+                    height: '100%',
+                    includeFontPadding: false,
+                    textAlign: 'center',
+                    textAlignVertical: 'center',
+                }),
                 fontFamily: font.name,
                 color: props.tintColor ?? DEFAULT_SYMBOL_COLOR,
                 fontSize: size,
diff --git a/src/SymbolView.tsx b/src/SymbolView.tsx
index 3df5fd5299da15f9e923c3fc0f0870bcca05be0e..e4e38502688b58012a151f40df04068253cd360f 100644
--- a/src/SymbolView.tsx
+++ b/src/SymbolView.tsx
@@ -40,7 +40,15 @@ export function SymbolView(props: SymbolViewProps): JSX.Element {
   return (
     <View style={style}>
       <Text
+        allowFontScaling={Platform.OS === 'android' ? false : undefined}
         style={{
+          ...(Platform.OS === 'android' && {
+            width: '100%',
+            height: '100%',
+            includeFontPadding: false,
+            textAlign: 'center',
+            textAlignVertical: 'center',
+          }),
           fontFamily: font.name,
           color: props.tintColor ?? DEFAULT_SYMBOL_COLOR,
           fontSize: size,
